incubator/elastic-stack/requirements.lock

```diff
@@ -1,24 +1,24 @@
 dependencies:
 - name: elasticsearch
   repository: https://kubernetes-charts-incubator.storage.googleapis.com/
-  version: 1.7.2
+  version: 1.8.1
 - name: kibana
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 0.13.1
+  version: 0.14.6
 - name: logstash
   repository: https://kubernetes-charts-incubator.storage.googleapis.com/
-  version: 0.6.3
-- name: fluentd
-  repository: https://kubernetes-charts-incubator.storage.googleapis.com/
-  version: 0.1.4
+  version: 0.9.3
+- name: fluentd-elasticsearch
```

Comment:
Why the change?
fluentd and fluentd-elasticsearch are two different things.

Reply:
Why different? As I understood, fluentd deploys the single instance of fluentd, and fluentd-elasticsearch deploys the DaemonSet with fluentd on each node. Also, it has configuration to gather logs from all containers on node.

Single fluentd itself is not helpful while deploying the elastic-stack, IMHO. What I would want from this chart is the working EFK for all my containers.

Reply:
So maybe we need options to both, actually.

Reply:
Look at the two charts.
We use the fluentd Chart for ingesting external logs into Elastic-Stack in one cluster and we also use fluentd-elasticstack to ingest container logs into elastic-stack into a different cluster.

Reply:
my point is that, in order to cover most use cases for elastic-stack, we would need to have fluentd AND fluentd-elasticsearch subcharts.

Actually, I already do this with custom charts. Doing it from upstream would be nice.

Reply:
Yeah, you right. I'll change PR to include both cases

Reply:
Sounds good to me.